jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/GradleLayerConfigurations.java

```diff
@@ -130,14 +133,31 @@ private static JavaLayerConfigurations getForNonWarProject(
     }
 
     // Adds dependency files.
+    List<String> duplicates =
```

Comment:
Does this ensure reproducibility of which is deemed the duplicate?

Reply:
It should just mark all duplicates with file size. No discrimination.

Reply:
Yes, this collects/renames all duplicates, e.g. with fileA, fileB, fileC duplicates, instead of the original plan of renaming to fileA, fileB-size, fileC-size, it's now fileA-size, fileB-size, fileC-size.